src/Aspire.Hosting.AppHost/Aspire.Hosting.AppHost.csproj

```diff
@@ -17,6 +17,7 @@
 
   <ItemGroup>
     <None Include="**/*.props;**/*.targets;AspireAppHostConfiguration.json" Pack="true" PackagePath="%(RecursiveDir)%(Filename)%(Extension)" />
```

Comment:
will the `in.targets` file also be globbed into the nuget package?

Reply:
Seems it isn't when I tried locally

Reply:
Local build showing what is in the package now:

<img width="768" alt="image" src="https://github.com/user-attachments/assets/a9b9da83-b89f-423d-a148-f2becf84312e" />